### ISSUE TITLE
Breaks and never recovers when the odrive is disconnected

### ISSUE THREAD

**Issue by dreusel** [zauberzeug/odrive-gui#4]
When the odrive is disconnected (ie power removed which in my case is sometimes needed because it goes in an error state after saving the config) the terminal output shows an error and then goes into an endless repetitive cycle from which it never recovers, also (specifically) not when reconnecting the ordive:
```
11:03:32.312403110 [LEGACY_OBJ] protocol failed with 3 - propagating error to application
ERROR base_events: Task exception was never retrieved
future: <Task finished name='Task-3' coro=<loop() done, defined at /usr/local/lib/python3.9/site-packages/nicegui/binding.py:17> exception=ObjectLostError('the object disappeared')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/nicegui/binding.py", line 24, in loop
    value = transform(getattr(source_obj, source_name))
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 638, in __get__
    return self._get_obj(instance).read()
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 581, in __call__
    return run_coroutine_threadsafe(self._libfibre.loop, lambda: self.__call__(*args))
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 296, in run_coroutine_threadsafe
    return future.result()
  File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 446, in result
    return self.__get_result()
  File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
    raise self._exception
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 291, in func_async
    result = await result
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 548, in async_call
    tx_buf, rx_chunk, is_closed = await agen.asend((tx_buf, self._rx_size - len(rx_buf), True))
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 470, in asend
    return await self.__anext__()
  File "/usr/local/lib/python3.9/site-packages/odrive/pyfibre/fibre/libfibre.py", line 499, in __anext__
    raise _get_exception(status)
fibre.libfibre.ObjectLostError: the object disappeared
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/nicegui/timer.py", line 41, in do_callback
    return callback()
  File "/app/controls.py", line 49, in <lambda>
    f'{axis.motor.current_control.Iq_measured * axis.motor.current_control.v_current_control_integral_q:.1f} W'))
AttributeError: 'EmptyInterface' object has no attribute 'motor'
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/nicegui/timer.py", line 41, in do_callback
    return callback()
  File "/app/controls.py", line 49, in <lambda>
    f'{axis.motor.current_control.Iq_measured * axis.motor.current_control.v_current_control_integral_q:.1f} W'))
AttributeError: 'EmptyInterface' object has no attribute 'motor'
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/nicegui/timer.py", line 39, in do_callback
    return await callback()
  File "/app/controls.py", line 144, in vel_push
    vel_plot.push([datetime.now()], [[axis.controller.input_vel], [axis.encoder.vel_estimate]])
```

It would be great if the application could resume when the odrive reconnects.

**Comment by dreusel** [comment]
I'm working on a fix for this.

**Comment by falkoschindler** [comment]
@dreusel Thanks, that would help a lot! 😀

**Comment by dreusel** [comment]
This should be fixed with #10
